src/Dofs/ConstraintHandler.jl

```diff
@@ -239,17 +236,32 @@ end
 
 Add the `AffineConstraint` to the `ConstraintHandler`.
 """
-function add!(ch::ConstraintHandler, newac::AffineConstraint)
-    # Basic error checking
-    for ac in ch.acs
-        (ac.constrained_dof == newac.constrained_dof) &&
-            error("Constraint already exist for dof $(ac.constrained_dof)")
-        any(x -> x.first == newac.constrained_dof, ac.entries) &&
-            error("New constrained dof $(newac.constrained_dof) is already used as a master dof.")
-    end
-    push!(ch.acs, newac)
-    push!(ch.prescribed_dofs, newac.constrained_dof)
-    push!(ch.inhomogeneities, newac.b)
+function add!(ch::ConstraintHandler, ac::AffineConstraint)
+    add_prescribed_dof!(ch, ac.constrained_dof, ac.b, isempty(ac.entries) ? nothing : ac.entries)
```

Comment:
Here we lose information about this coming from an affine constraint. That is probably mostly fine, but in `update!` we use this to see if the inhomogeneity should be updated or not...
```suggestion
    add_prescribed_dof!(ch, ac.constrained_dof, ac.b, ac.entries)
```